### DISCUSSION TITLE
Docker 更新依然显示旧版本

### DISCUSSION THREAD

**Discussion by Jiematcha** [xfgryujk/blivechat#177] [提问] [answered]
Docker 1.8.2 更新 1.9.0 网页依然显示 1.8.2
Shift F5 等依然无效

rm -r /var/lib/docker/volumes/blivechat-data/

后重新安装 成功安装 1.9.0

**Comment by xfgryujk** [comment, marked as answer]
Docker是这样的，已经创建了卷再次run镜像就不会修改卷的内容了
